src/connect-light-node.md

```diff
@@ -18,14 +18,15 @@ send-messages-loop \
 --pubsub-topic=/waku/2/rs/66/0 \
 --cluster-id=66 \
 --lightpush-peer=/ip4/10.2.0.16/tcp/60000/p2p/16Uiu2HAmAA99YfoLitSXgY1bHaqjaTKhyrU4M4y3D1rVj1bmcgL8 \
---message-every-secs=5
+--message-every-secs=5 \
+--epoch-size-secs=60
```

Comment:
Don't mind about the number but can we be consistent with the epoch size?

In deployment section we use `RLN_RELAY_MSG_LIMIT` [here](https://waku-org.github.io/waku-simulator/deploy-waku-network.html)

and in other places we use `--rln-relay-epoch-sec=1` as well.

So would suggest to set =1 here. or use the same number all over the book.

The examples present in the book should all run against the same instance of waku simulator. And if we set this to 60, I believe that wont be the case.

Reply:
all epoch values set to `1` in https://github.com/waku-org/waku-simulator/pull/68/commits/bcc6ef80362970bad1a3b5bef2f124f981a53eea